docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -52,15 +100,15 @@ cache:
    .. code:: bash
 
       cd $singularity_image_root
-      image="determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-9119094"
+      image="determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-096d730"
       singularity pull $image docker://$image
 
-#. Otherwise, from an internet-connected system, download the desired image using the Singulartity
+#. Otherwise, from an internet-connected system, download the desired image using the Singularity
```

Comment:
Not sure if "internet" needs to be capitalized.

Reply:
No-one else is sure either :-)

https://grammarist.com/style/internet-capitalization/